File: SoloLearn/Gotham City/GothamCity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int nb = Convert.ToInt32(Console.ReadLine());
            if (nb < 5) 
                Console.WriteLine("I got this!");
            else if (nb < 11)
                Console.WriteLine("Help me Batman");
            else
                Console.WriteLine("Good Luck out there!");
        }
    }
}

File: SoloLearn/Code Coach/Gotham City/GothamCity.cpp
#include <iostream>
using namespace std;

int main() {
    int nb = 0;
    cin >> nb;
    if (nb < 5)
        cout << "I got this!";
    else if (nb < 11)
        cout << "Help me Batman";
    else
        cout << "Good Luck out there!";
    return 0;
}